### PR TITLE
Landing page without enum

### PR DESCRIPTION
**Pull request by mgunnhawkins**
1.  Removed Enum
2. Changed detourRadius to Integer
3. Changed Thymeleaf form to number instead of dropdown.
4. Data is saved as a number in the database!!!

### CODE REVIEW

File: src/main/java/com/roadtriprunner/RoadTripRunner/models/Location.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//package com.roadtriprunner.RoadTripRunner.models;
```
Comment (valerietonsor) [inline src/main/java/com/roadtriprunner/RoadTripRunner/models/Location.java]: let's delete this if not being used.

File: src/main/resources/templates/index.html [hunk 2]
```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org/">
-<head>
```
Comment (valerietonsor) [inline src/main/resources/templates/index.html]: make sure we are using the head fragment

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
-    <meta charset="UTF-8"/>
-    <title></title>
-</head>
+<head th:replace="fragments :: head"></head>
 <body>
-    <h1>Hello, World!</h1>
```
Comment (valerietonsor) [inline src/main/resources/templates/index.html]: delete

**Review by valerietonsor — CHANGES_REQUESTED** [review]
MELANIE. You did such great work on this. I know it was frustrating... you had so much more patience with the enum than I did. Great freaking work. Just made a few comments about things that can be removed, but functionality is AMAZING.

**Review by krystalwhite — APPROVED** [review]
Impressive work integrating all the pieces!

**Review by valerietonsor — APPROVED** [review]

**Review by krystalwhite — APPROVED** [review]

**Pull request merged**